### ISSUE TITLE
🧩 FEATURE REQUEST: Re-fetch new articles only after some idle time

### ISSUE THREAD

**Issue by aranwe** [dailydotdev/daily#375]
### Description

When I open new tab and type in URL I quite often glance at an article heading and realize just as I am hitting enter that I actually wanted to read it. 
After I open new tab daily.dev re-fetches the articles and the one I actually wanted to read disappears (or is in different position on the page making it difficult to find it).

### Solution

I would like daily.dev to display new content only after some idle time (5minutes?). 
I.e. if I open subsequently 10 new tabs each should have the same articles in exactly the same order.
As a nice to have daily.dev could actually pre-fetch the new content and then show it once the idle timer triggers making the experience lightning fast, preventing the flash of empty page we currently have :) 

### Alternate Solutions

NA

**Comment by idoshamun** [comment]
Hi @aranwe,

Thanks for the suggestion actually we have a cache mechanism in place that works a bit differently than what you suggest.
We set the TTL for 3 minutes for the feed cache the moment we generate a new feed but don't wait for IDLE times.
The tricky part with a New Tab extension is that IDLE is hard to define because opening new tabs is just a so common action.

I do agree that we can keep the content in the local cache as well for better performance ⚡
